README.md

```diff
@@ -11,28 +11,41 @@
 
 Besu is an Apache 2.0 licensed, MainNet compatible, Ethereum client written in Java.
 
+## Useful Links
+
+* [Besu Docs](http://besu.hyperledger.org/)
+* [Besu Jira](https://jira.hyperledger.org/projects/BESU/issues)
```

Comment:
`Besu Jira` and`contributing` links can be reused as a ref too. Having the link written only once prevents typos and the need for reviewers to check that teach link are valid because visual inspection is not enough and they have to follow the link, so double work if there's the link written twice.

Reply:
I'm not sure I follow, Both of those are only listed once each.